convert_zdf.py

```diff
@@ -0,0 +1,150 @@
+"""Convert ZDF point cloud to the available formats.
+
+Example: $ python -u convert_zdf.py --ply Zivid3D.zdf
+
+Available formats:
+    .ply - Polygon File Format
+    .csv,.txt - [X, Y, Z, r, g, b, Contrast]
+    .png,.jpg,.bmp,.tiff - 2D RGB image
+"""
+
+import argparse
+from pathlib import Path
+import numpy as np
+import cv2
+import zivid
+
+
+def _options():
+    parser = argparse.ArgumentParser(
+        description="Convert from ZDF to your preferred format\
+            \nExample:\n\t $ python -u convert_zdf.py --ply Zivid3D.zdf",
```

Comment:
why the "-u" argument? Is this critical for this to work?

Reply:
I believe I had it in there to force flush stdout. It was then useful for multiple conversions to show progress.